redash/query_runner/cass.py

```diff
@@ -77,10 +94,14 @@ def run_query(self, query, user):
             if self.configuration.get('username', '') and self.configuration.get('password', ''):
                 auth_provider = PlainTextAuthProvider(username='{}'.format(self.configuration.get('username', '')),
                                                       password='{}'.format(self.configuration.get('password', '')))
-                connection = Cluster([self.configuration.get('host', '')], auth_provider=auth_provider, protocol_version=3)
+                connection = Cluster([self.configuration.get('host', '')],
+                                     auth_provider=auth_provider,
+                                     protocol_version=self.configuration.get('protocol', ''),
```

Comment:
To be consistent with previous versions (and the stated default value), let's change this to: `self.configuration.get('protocol', 3)`.